sunholmexp.py

```diff
@@ -391,6 +410,25 @@ class LevelingUpPlayer:
     leveled_up: bool = False
     quest_log_bonus_gold: int = 0
 
+# Total amount of experience needed to make a character gain level_change levels instantly
+# If preserve is set, their percentage progress from their current level carries over
+def exp_after_bonus_leveling(current_exp: int, level_change: int = 1, preserve: bool = False) -> int:
```

Comment:
I was actually hoping this pr would get merged first and then @g-harel would have the privilege of handling that deduplication :^). Otherwise we have to base off each others forks and it gets a bit messier

Reply:
You honor me too much with your generosity @mbwangfpdc

Reply:
If only this one got to patch first.

Reply:
Such is life